listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -14,6 +14,28 @@
 from listenbrainz.db.exceptions import DatabaseException
 from spotipy import SpotifyException
 from werkzeug.exceptions import BadRequest, InternalServerError, ServiceUnavailable
+from brainzutils.mail import send_mail
+from brainzutils import musicbrainz_db
+from brainzutils.musicbrainz_db import editor as mb_editor
+
+
+def notify_error(musicbrainz_row_id, error):
+    """ Notifies specified user via email about error during Spotify import.
+
+    Args:
+        musicbrainz_row_id (int): the MusicBrainz row ID of the user
+        error (str): a description of the error encountered.
+    """
+    user_email = mb_editor.get_editor_by_id(musicbrainz_row_id)['email']
+    text = ('Hi, we encountered an error while importing your listens from Spotify.'
+            'The error was as follows: "%s" '
+            'Please take a look at your profile page for more information.\n'
```

Comment:
Can we include a link to their profile in the mail?

Reply:
What email address is the email being sent from?

Reply:
We can set it here: https://github.com/metabrainz/listenbrainz-server/pull/467/files#diff-1e4b2ed41bf1b80350b586a316431d06R141

I have it currently as `listenbrainz.org` so it tries to send the mail from `noreply@listenbrainz.org`, I intended to ask if there's more setup needed for the email.

Reply:
noreply is great -- consistent with our other services. That's all I wanted to hear. :)